packages/shared/src/modules/projects/libs/types/project-get-all-item-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type ProjectGetAllItemResponseDto = {
```

Comment:
I think better to use plural `-Items-` instead of singular `-Item-`
```suggestion
type ProjectGetAllItemsResponseDto = {
```
Do not forget about file name and other places it's used

Reply:
'getAll' gets many items. And in this dto we have 1 item from all items that we got. 
Literally "one item from method 'getAll' ".
Hope i explained clear

Reply:
yes, we use Item here because it's just 1 item

Reply:
Oh, I didn't notice that. Then sure